sympy/functions/elementary/complexes.py

```diff
@@ -896,18 +1064,32 @@ class periodic_argument(Function):
     logarithm. That is, given a period P, always return a value in
     (-P/2, P/2], by using exp(P*I) == 1.
 
+    Examples
+    ========
+
     >>> from sympy import exp, exp_polar, periodic_argument, unbranched_argument
     >>> from sympy import I, pi
     >>> unbranched_argument(exp(5*I*pi))
     pi
     >>> unbranched_argument(exp_polar(5*I*pi))
     5*pi
-    >>> periodic_argument(exp_polar(5*I*pi), 2*pi)
-    pi
-    >>> periodic_argument(exp_polar(5*I*pi), 3*pi)
-    -pi
-    >>> periodic_argument(exp_polar(5*I*pi), pi)
-    0
+
+    Parameters
+    ==========
+
+    ar : Expr
+        A polar number.
```

Comment:
This can use the examples like below.
I think that you can make make examples that fixes `period` to `2pi` and try out different `exp_polar`.
Although this function can take more general inputs, I think that the standard usage would be using exp_polar with multiples of `pi`

Reply:
Done.